slitlessutils/core/preprocess/crrej/tests/test_drizzle.py

```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+from astroquery.mast import Observations
+
+from slitlessutils.core.preprocess.crrej.drizzle import drizzle
+from slitlessutils.examples.wr96 import WR96_DRIZZLE_PARAMS
+
+
+def test_wr96_drizzle():
+    '''
+    Runs the drizzle step embedded in the wr96 example
+    '''
+    uris = {
```

Comment:
There are two problems here.  
1) all of these files are not grism images.  only 'jdql01jpq_flc.fits' and 'jdql01jxq_flc.fits' are.  And these variables are already set in DATASETS[GRATING] in the preamble. And it makes no sense to drizzle together data that are not the same filter

2) probably dont' want call the variable 'uris'.  The detector is uvis, that might lead to confusion to someone peering in at us.

Reply:
Done in https://github.com/spacetelescope/slitlessutils/pull/27/commits/2eee695a57c442ebc7e86a451801039a42e6deb4